src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -14154,6 +14154,8 @@ def test_aks_disable_ssh(self, resource_group, resource_group_location):
         # delete
         self.cmd('aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    # live only due to the case is likely to fail in playback mode
+    @live_only()
```

Comment:
Just out of curiosity, I would like to ask why the playback mode fails

Reply:
cannot tell for sure, this may be related to our special handling of an lro. Compared with normal lro handled by SDK, in this case, we will get the resource immediately after the lro is initiated, and then poll the lro until it's done. This may cause vcr to make occasional wrong judgments? like taking the first get by us as the final get in the polling process
https://dev.azure.com/azclitools/public/_build/results?buildId=175979&view=logs&j=155127ab-52f1-52e7-0e45-625d9d7884a4&t=4bb67950-ef03-52d4-44f5-d1a8cd56c919&l=2398

Reply:
> No match for the request (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.Network/virtualNetworks/cliakstest000002/subnets/nodeSubnet/providers/Microsoft.Authorization/roleAssignments/73dbacd5-ad82-41d4-b546-bab9b201ddc7?api-version=2022-04-01>) was found.
Found 1 similar requests with 1 different matcher(s) :

>1 - (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.Network/virtualNetworks/cliakstest000002/subnets/nodeSubnet/providers/Microsoft.Authorization/roleAssignments/bec87765-2d5b-4fe4-b023-335d91697720?api-version=2022-04-01>).

This seems to be a GET request that was not recorded. Could you try recording the yaml file corresponding to this test in live mode again?

Reply:
I did, but it didn't help. Besides the test failure is not deterministic, but occasional.